src/Compilers/Core/Portable/Emit/EditAndContinue/DeltaMetadataWriter.cs

```diff
@@ -228,6 +230,17 @@ public void GetMethodTokens(ICollection<MethodDefinitionHandle> methods)
             }
         }
 
+        /// <summary>
+        /// Return tokens for all modified debuggable types.
+        /// </summary>
+        public void GetTypeTokens(ICollection<TypeDefinitionHandle> types)
```

Comment:
>GetTypeTokens [](http://example.com/codeflow?start=20&length=13)

perhaps rename to `GetUpdatedTypeHandles` - it's not immediately clear what types does this return. (same for GetMethodTokens)